src/LoginPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Container, FormGroup, Label, Input, Button } from "reactstrap";
+
+function LoginPage() {
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const forms = [
+    { text: "Email", name: "email", callback: setEmail },
+    { text: "Password", name: "password", callback: setPassword }
+  ];
+
+  function formEmpty() {
+    return email.length && password.length;
+  }
+
+  return (
+    <Container>
+      <form onSubmit={(e) => { e.preventDefault(); }}>
+        {forms.map((x, index) => {
+          return (
+            <FormGroup key={index}>
+              <Label>{x.text}</Label>
+              <Input
+                autoFocus
+                name={x.name}
+                type={x.name}
+                onChange={(e) => x.callback(e.target.value)}
```

Comment:
change x.callback according to the fix above

Reply:
Done.